benchmarks/bloaty_fuzz_target/benchmark.yaml

```diff
@@ -18,3 +18,4 @@ fuzz_target: fuzz_target
 project: bloaty
 unsupported_fuzzers:
   - klee
+  - symcc_afl
```

Comment:
this is disabled on too many benchmarks, what is the issue here ? ideally we want stuff on work on most coverage based benchmarks (can exclude 1-2) and can be several bug based ones.

Reply:
I will take a look at this in more depth early next week!